### PR TITLE
making changes to test alerts

### PR DESCRIPTION
**Pull request by dibyanshuMoj**

### CODE REVIEW

**Review by KalyanaVadlamani — APPROVED** [review]

File: charts/generic-aws-prometheus-alerts/values.yaml [hunk 1]
```diff
@@ -32,10 +32,10 @@ rdsAlertsCPUThresholdMinutes: 2
 # https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html#RDS_Limits.MaxConnections then provides
 # calculations for each database on the maximum number of database connections so for a db.t3.small database it will be
 # LEAST({2Gib/9531392}, 5000) = LEAST({2 * 1024 * 1024 * 1024/9531392}, 5000) = LEAST(225, 5000) = 225.
-rdsAlertsConnectionThreshold: 180
+rdsAlertsConnectionThreshold: 5
```
Comment (babuharidass11) [inline charts/generic-aws-prometheus-alerts/values.yaml:35]: why these connection count are low.?

File: charts/generic-aws-prometheus-alerts/values.yaml [hunk 2]
```diff
@@ -20,7 +20,7 @@ additionalRuleLabels: {}
 #   database-name-id: "Database nice display name"
 #   database-name-2-id: "Database nice display name 2"
 
-rdsAlertsCPUThreshold: 5
+rdsAlertsCPUThreshold: 1
```
Comment (babuharidass11) [inline charts/generic-aws-prometheus-alerts/values.yaml:23]: why these values are low.?

File: charts/generic-aws-prometheus-alerts/values.yaml (hunk 1 above)
Comment (dibyanshuMoj) [reply charts/generic-aws-prometheus-alerts/values.yaml:35]: To trigger the alerts, it was set to 5. Will revert this once the alert trigger is tested

File: charts/generic-aws-prometheus-alerts/values.yaml (hunk 2 above)
Comment (dibyanshuMoj) [reply charts/generic-aws-prometheus-alerts/values.yaml:23]: To trigger the alerts, it was set to 5. Will revert this once the alert trigger is tested

File: charts/generic-aws-prometheus-alerts/values.yaml (hunk 1 above)
Comment (babuharidass11) [reply charts/generic-aws-prometheus-alerts/values.yaml:35]: this is generic alert and test alert values should be changed to terraform hmpps-service-catalogue (dev namespace) deployment

File: charts/generic-aws-prometheus-alerts/values.yaml (hunk 2 above)
Comment (babuharidass11) [reply charts/generic-aws-prometheus-alerts/values.yaml:23]: this is generic alert and test alert values should be changed to terraform hmpps-service-catalogue (dev namespace) deployment

**Pull request merged**